apps/settings/api/chat.py

```diff
@@ -101,6 +101,9 @@ def get_permissions(self):
     def filter_default_prompts(self):
         lang = self.request.LANGUAGE_CODE
         default_prompts = DefaultChatPrompt.get_prompts(lang)
+        if not default_prompts:
+            return []
+
         search_query = self.request.query_params.get('search')
         search_query = search_query or self.request.query_params.get('name')
         if not search_query:
```

Comment:
No significant changes identified. The only difference is adding an else block to ensure 'default_prompts' variable is assigned value instead of returning empty list when it's not present. No other issues or improvements needed.

English Translation: